py/tests/generic_test_utils.py

```diff
@@ -23,24 +23,28 @@
 
 
 def unique_cloud_name(script_name):
-    return str(script_name[:-3].replace("/", "_"))+str(randrange(65536))
+    return str(script_name[:-3].replace("/", "_")) + str(randrange(65536))
+
 
 def locate(file_name):
     if os.path.isfile("/home/0xdiag/" + file_name):
         return os.path.abspath("/home/0xdiag/" + file_name)
     else:
         return os.path.abspath("../examples/" + file_name)
 
-def cluster_mode(spark_conf = None):
+
+def cluster_mode(spark_conf=None):
```

Comment:
I would prefer the original version with spaces or is it more frequent like this in Python?